docs/user/PostgreSQL.md

```diff
@@ -0,0 +1,60 @@
+# PostgreSQL
+
+* [PostgreSQL](#postgresql)
+  * [Use IAC To Create an External PostgreSQL Database Cluster](#use-iac-to-create-an-external-postgresql-database-cluster)
+  * [Post Data Transfer Steps for viya4-deployment (Experimental)](#post-data-transfer-steps-for-viya4-deployment-experimental)
+    * [Crunchy Data 5](#crunchy-data-5)
+    * [Crunchy Data 4](#crunchy-data-4)
+
+## Use IAC To Create an External PostgreSQL Database Cluster
+
+To use the IAC project to create an external PostgreSQL database cluster, refer to the IAC project link below that corresponds to your cloud environment. Each link goes to provider-specific PostgreSQL database cluster configuration examples.
+
+
+**Note**: Before using IAC to create a new external PostgreSQL database cluster, SAS recommend that you follow the steps to [Stop a SAS Viya Platform Deployment](https://documentation.sas.com/?cdcId=sasadmincdc&cdcVersion=v_044&docsetId=calchkadm&docsetTarget=p17xfmmjjkma1dn1b5dcx3e5ejxq.htm#p0butgo7gtfyi0n14umtfv0voydt). After the external PostgreSQL database cluster has been created by IAC, follow the steps to [Start a SAS Viya Platform Deployment](https://documentation.sas.com/?cdcId=sasadmincdc&cdcVersion=v_044&docsetId=calchkadm&docsetTarget=p17xfmmjjkma1dn1b5dcx3e5ejxq.htm#p0butgo7gtfyi0n14umtfv0voydt).
```

Comment:
SAS recommends

[add an s to "recommend"]

Reply:
Update made